### DISCUSSION TITLE
Add user labels/predictions to suggestions

### DISCUSSION THREAD

**Discussion by talmo** [talmolab/sleap#1418] [Ideas]
In large projects with many labels or predictions, it can be difficult to keep track of all of the labeled frames, especially after multiple rounds of labeling, when you have multiple annotators, and other non-standard workflows.

The suggestions table helps when it's used, but for some cases there may be labels that aren't included in that table.

#736 would help, but until we converge on a good solution for browsing through all labels, a good workaround would be to have the ability to _manually_ add frames to this table for ease of labels management.

This could work by having options under the **Labels** menu to "Add user-labeled frames to suggestions" and "Add predicted frames to suggestions".

The logic would basically be:

```py
for lf in labels:
    if lf.has_user_instances:  # or lf.has_predicted_instances
        labels.add_suggestion(lf.video, lf.frame_idx)
```

**Comment by xlkg** [comment]
Hello,
We would be very interested in this functionality, ie to be able to add to the label-suggestions list, a list of frames to label (eg identified as mislabeled by the prediction). An additional choice in the Labeling suggestions menu could be "Add user-labeled frames from file", as proposed here. Could you let us know if this was finally implemented, or if it could be in a near future?
Best, Xavier
